app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
**You did a great job!** 

- [ ] However, in your comments, I was not able to find `new` as your posts have. Kindly add new to your comments `controllers`, `routes`.